$NetBSD: patch-libraries_text_text.cabal,v 1.2 2023/10/09 02:49:52 pho Exp $

This is a temporary workaround for aarch64 codegen bug in 9.2.1. Once we
update our bootkit for aarch64 we can remove this.
https://gitlab.haskell.org/ghc/ghc/-/issues/20594

--- libraries/text/text.cabal.orig	2023-08-21 03:50:41.000000000 +0000
+++ libraries/text/text.cabal
@@ -127,10 +127,6 @@ library
   if (arch(aarch64) || arch(arm)) && impl(ghc == 8.10.*)
     build-depends: base < 0
 
-  -- Subword primitives in GHC 9.2.1 are broken on ARM platforms.
-  if (arch(aarch64) || arch(arm)) && impl(ghc == 9.2.1)
-    build-depends: base < 0
-
   -- NetBSD + GHC 9.2.1 + TH + C++ does not work together.
   -- https://gitlab.haskell.org/ghc/ghc/-/issues/22577
   if flag(simdutf) && os(netbsd) && impl(ghc < 9.4)
